deploy/Offline-version.json

```diff
@@ -0,0 +1 @@
+{"version": "0.0.1"}
```

Comment:
This does not match the version specified in `defaults.py`. Can the settings file load this in directly, to avoid information getting out of sync?